src/app/shared/consts.ts

```diff
@@ -59,6 +67,23 @@ export const LENGTH_OF_STAY: Option[] = [
   { code: AccommodationOffer.LengthOfStayEnum.Longer, label: LengthOfStayLabels.LONGER },
 ];
 
+export const LAW_HELP_MODES: Option[] = [
```

Comment:
to chyba nie będzie potrzebne, możemy spiąć gotowe enumy z core/api i spiac bezposrednio z tlumaczneiem.
MODE_ENUM = Object.values(JobOfferDefinitionDTO.ModeEnum); - zobacz w kategorii help-care

Reply:
Zrobione.